build.sbt

```diff
@@ -2517,6 +2517,7 @@ lazy val `engine-runner` = project
     commands += WithDebugCommand.withDebug,
     inConfig(Compile)(truffleRunOptionsSettings),
     libraryDependencies ++= Seq(
+      "org.graalvm.sdk"         % "launcher-common"         % graalMavenPackagesVersion,
```

Comment:
Using `launcher-common` is not easy. It is not a nicely designed API. Rather an abstraction of common functionality found among various Truffle language launchers. Some things aren't even possible/easy - for example get list of unprocessed arguments.